src/main/java/com/revature/controllers/ModuleController.java

```diff
@@ -87,6 +85,11 @@
         return ResponseEntity.ok(jsonModules);
     }
 	
+//	// return list of  modules attached to a given curriculum
+//	@GetMapping("/{id}/curriculum")
+//    public ResponseEntity<Set<Module>> getModuleByCurriculumId(@PathVariable int id) {
+//		return ResponseEntity.ok(moduleService.getModulesByCurricumId(id));
+//	}	
```

Comment:
why include this?

Reply:
I include it before I implemented another method on the content controller that gets the modules and content by the curriculum. id it should be removed now.